localization/german/culture/ek_culture_history_loc_l_german.yml

```diff
@@ -1,50 +1,45 @@
-    imperial_history_loc:0 "Developed out of [CULTURE.GetName] tribes under [TARGET_CULTURE.GetName] rule"
-    dunmer_history_loc:0 "Born from Velothi who submitted to the Tribunal responsible for their curse"
-    ashlander_history_loc:0 "Born from Velothi who denounced the Tribunal responsible for their curse"
-    orc_history_loc:0 "Changed from their Aldmer forms after Trinimac's transformation"
-    iron_orc_history_loc:0 "Renounced their peaceful ways after Nedic aggression"
-    redguard_history_loc:0 "Descended from the [CULTURE.GetName] [ragada|E] settlers who invaded the Deathlands"
-    crown_history_loc:0 "Scions of the Old [CULTURE.GetName] Na-Totambu high nobility"
-    yokudan_history_loc:0 "Survivors of the sinking of Old Yokuda"
-    abecean_history_loc:0 "Descended from the [CULTURE.GetName] Tavan [ragada|E] led by Prince Hubalajad"
-    argonian_history_loc:0 "Blessed with sapience by the Hist during the [merethic_era|E]"
-    mathmeldi_history_loc:0 "[CULTURE.GetCollectiveNoun] driven from their homes in Ald-Cyrod"
-    barsaebic_history_loc:0 "Descended from [enc_aedric|E] [CULTURE.GetName] refugees of the [narfinsel_schism|E]"
-    dynari_history_loc:0 "Descended from the Nenalata [CULTURE.GetName]"
-    bosmer_history_loc:0 "Formed by Y'ffre from the primordial Ooze"
-    strident_history_loc:0 "Formed from Gold Coast [CULTURE.GetCollectiveNoun] under [TARGET_CULTURE.GetName] influence" 
-    blackwood_history_loc:0 "Kept the old ways of the [CULTURE.GetName]"
-    rimmenese_history_loc:0 "[CULTURE.GetName] traders settled in the Rim Territories" 
-    daedra_history_loc:0 "Interlopers from the Planes of Oblivion"
-    maormer_history_loc:0 "Cast out from fabled [aldmeris|E] in the [dawn_era|E]"
-    beyonder_history_loc:0 "[CULTURE.GetName] adventurers and pirates far from their mythical home of Pyandonea"
-    skaal_history_loc:0 "Diverged from their Mainland cousins after years of isolation"
-    nedic_history_loc:0 "Descended from unknown [CULTURE.GetName] tribal groups"
-    yespest_history_loc:0 "Developed out of [CULTURE.GetName] tribes under [TARGET_CULTURE.GetName] rule"
-    ojel_history_loc:0 "Developed out of [CULTURE.GetName] tribes under [TARGET_CULTURE.GetName] rule"
-    karth_history_loc:0 "Descended from the Old [CULTURE.GetCollectiveNoun]"
-    grath_history_loc:0 "Progeny of [CULTURE.GetCollectiveNoun] who settled the Great Grahtwoods"
-    silvenar_history_loc:0 "Progeny of [CULTURE.GetCollectiveNoun] who settled in what is now Silvenar"
-    imga_history_loc:0 "Nearly as old as Nirn itself, no one really knows their true origin."
-
-
-
-
-
+l_german:
+    mystery_history_loc:0 "Die Herkunft dieses Volkes ist von Mysterien umwogen"
+    altmer_history_loc:0 "Entstammt den sagenumwogenen [aldmeris|E] am [dawn_era|E]"
+    direnni_history_loc:0 "Entstammt den [CULTURE.GetName] Verwandten, die sich in Balfiera in der Zeit der [first_era|E] niederließen"
```

Comment:
Entstammt den [CULTURE.GetName] Verwandten, die sich **auf** Balfiera in der [first_era|E] niederließen